src/main/scala/it/pagopa/interop/backendforfrontend/service/types/AttributeRegistryServiceTypes.scala

```diff
@@ -5,14 +5,18 @@ import it.pagopa.interop.backendforfrontend.model.AttributeKind.{CERTIFIED, DECL
 import it.pagopa.interop.backendforfrontend.model.{
   Attribute,
   AttributeKind,
-  AttributesResponse,
   AttributeSeed,
-  CertifiedAttribute
+  AttributesResponse,
+  CertifiedAttribute,
+  DeclaredAttribute,
+  VerifiedAttribute
 }
 
 object AttributeRegistryServiceTypes {
   type AttributeRegistryManagementInvoker = attributeregistrymanagement.client.invoker.ApiInvoker
   type AgreementProcessInvoker            = agreementprocess.client.invoker.ApiInvoker
+  type CatalogManagementInvoker           = catalogmanagement.client.invoker.ApiInvoker
+  type TenantManagementInvoker            = tenantmanagement.client.invoker.ApiInvoker
   type MgmtAttribute                      = attributeregistrymanagement.client.model.Attribute
   type MgmtAttributesResponse             = attributeregistrymanagement.client.model.AttributesResponse
   type MgmtAttributeKind                  = attributeregistrymanagement.client.model.AttributeKind
```

Comment:
Why put here all the type aliases? I expected that each type stays in a dedicated object.

Reply:
Not sure, I just kept the existing approach, but it can be revisited (#tomorrow)